### ISSUE TITLE
How did you get your Flutter Web PWA accepted to Adsense?

### ISSUE THREAD

**Issue by d-apps** [amirk3321/flutter_ad_manager_web#1]
Were you able to get your PWA accepted normally as if it were a regular website?

are impressions and clicks working fine?

Thanks.

**Comment by amirk3321** [comment]
Yes, the PWA was accepted as a regular website, and impressions and clicks are working fine. If you check the [Inkblot Art](https://inkblot.art) website, which We have developed entirely using Flutter technology, you will get an idea.

Live Preview:
<img width="1439" alt="Screenshot 2023-11-16 at 4 32 20 PM" src="https://github.com/amirk3321/web-socket-example/assets/10207753/f99aae3c-4931-449c-98d4-b7ba5d782277">
